p751toolbox/field_decl.go

```diff
@@ -34,9 +38,30 @@ func fp751Mul(z *fp751X2, x, y *Fp751Element)
 
 // Perform Montgomery reduction: set z = x R^{-1} (mod 2*p).
 // Destroys the input value.
+var fp751MontgomeryReduce func(z *Fp751Element, x *fp751X2)
```

Comment:
Comment above is wrong - this is a variable not a function. It doesn't perform anything